components/treeview/events.md

```diff
@@ -288,6 +294,281 @@ The `OnItemClick` event fires when the user clicks (or presses `Enter`) on an no
 }
 ````
 
+## OnItemDoubleClick
+
+The `OnItemDoubleClick` event fires as a response to the user double clicking on a TreeView node.
```

Comment:
Simpler

```suggestion
The `OnItemDoubleClick` event fires when the user double-clicks a TreeView node.
```

On a side note, we may mention what happens on touch devices. Does this event ever fire?

Reply:
Indeed, checking with Stamo if the behavior will be the same as in the Grid. Will update the doc tomorrow morning.

Reply:
Confirmed, OnItemDoubleClick fires on double tap for mobile devices. Updated.